empack/extend_js.py

```diff
@@ -19,23 +19,80 @@ def extend_logging(file_path):
             out_file.write(line)
 
 
-def hotfixes(files):
+def clapack_hotfix(files):
     for f in files:
         if 'clapack_all.so' in f['filename']:
             f['filename'] = 'clapack_all.so'
+
+
     return files
 
+def scipy_hotfix(files):
```

Comment:
I think we should keep those hacks in a separate file, e.g. `patches.py`. What do you think?

Reply:
sure, why not